Dockerfile

```diff
@@ -5,11 +5,8 @@ ENV XDG_CACHE_HOME=/opt/xdg
 
 RUN apk add curl tar bash --no-cache
 RUN set -ex \
-    && curl -sSL https://get.helm.sh/helm-v3.3.1-linux-amd64.tar.gz | tar xz \
+    && curl -sSL https://get.helm.sh/helm-v3.8.2-linux-amd64.tar.gz | tar xz \
     && mv linux-amd64/helm /usr/local/bin/helm \
-    && rm -rf linux-amd64 
-RUN apk add --virtual .helm-build-deps git make \
```

Comment:
The chart museum plugin is still needed if the user does not want to use OCI and instead wants to push a tar.gz file. version 0.9.0 conflicts with the new `helm push` command, but 0.10.0 renames it to `helm cm-push` so there should be no conflict. A simple if statement should suffice in entrypoint.sh

Actually I also think it's worthwhile to add the helm s3 plugin, so that this action would support `s3:` urls https://github.com/hypnoglow/helm-s3 but that's probably a different issue and PR.